server/config/options.go

```diff
@@ -105,8 +106,9 @@ var DefaultConfig = Config{
 		SampleRate: 0.01,
 	},
 	Server: ServerConfig{
-		Host: "0.0.0.0",
-		Port: 8081,
+		Host:      "0.0.0.0",
+		Port:      8081,
+		FDBDelete: false,
```

Comment:
This is not needed. It's false by default.

Reply:
I intentionally added it to make sure behavior is clear.